manifest/armv7l/l/lcms.filelist

```diff
@@ -1,19 +1,15 @@
-# Total size: 2640809
-/usr/local/bin/jpgicc
-/usr/local/bin/linkicc
-/usr/local/bin/psicc
-/usr/local/bin/tificc
-/usr/local/bin/transicc
```

Comment:
What happened to all the binaries?

Reply:
We don't need them... ChromeOS will handle color management internally. We just need the stub libraries.